contrib/spark/README.md

```diff
@@ -0,0 +1,26 @@
+# Kubeflow Spark Operator
```

Comment:
If you move it to apps, you don't need to have README like for other Kubeflow components: https://github.com/kubeflow/manifests/tree/master/apps
We already have all of the required docs here: https://www.kubeflow.org/docs/components/spark-operator/

Reply:
We can follow up in another PR. This is still an experiment.